test/test-distributed-work.jl

```diff
@@ -0,0 +1,27 @@
+using Distributed
+using Extrae
+
+addprocs_extrae(1)
+
+@everywhere using Cassette
+@everywhere using Extrae
+
+
+function random_sleep()
+    println("Worker started: ", myid())
+    sleep(rand((1,2,3,4,5)))
+    println("Worker woke up: ", myid())
+end
+
+function test_distributed_work()
+    @everywhere Extrae.init()
+    Cassette.overdub(Extrae.ExtraeCtx(), remote_do, Cassette.overdub, 2, Extrae.ExtraeCtx(), random_sleep)
+    # Cassette.overdub(Extrae.ExtraeCtx(), remote_do, Cassette.overdub, 3, Extrae.ExtraeCtx(), random_sleep)
+    # Cassette.overdub(Extrae.ExtraeCtx(), remote_do, Cassette.overdub, 4, Extrae.ExtraeCtx(), random_sleep)
+    sleep(10)
+    @everywhere Extrae.finish()
+end
+
+test_distributed_work()
+
+println("END TEST")
```

Comment:
This is not how Julia tests are supposed to work 😅 
But I think we can do something.

Reply:
I know, it is just a fast way for me to try things. We can put it somewhere else!!

Reply:
In that case, I like putting this kind of codes inside the `scripts/` folder. Do you mind moving it there?